specs/Device/Device/doc/spec.md

```diff
@@ -185,6 +185,19 @@ The data model is defined as shown below:
             -   Type: [DateTime](http://schema.org/DateTime)
     -   Optional
 
+-   `rssi` : Received signal strength indicator for a wireless enabled device.
+    It must be equal to `1.0` when the signal strength is maximum. `0.0` when
+    signal is missing. `null` when it cannot be determined.
```

Comment:
as NGSI-LD forbids null we are not allowing null in attributes. can you think of another way to convey "not known"?

Reply:
Would a negative value work ? Say -1?

Reply:
yes